src/worker.js

```diff
@@ -331,12 +332,29 @@ self.onmessage = (e) => {
       }
       constraint.uuid = uuid
       state.world.addConstraint(constraint)
+
+      if (!isNaN(maxMultiplier)) {
```

Comment:
Please don't use isNaN

Reply:
OK. Would a truthy check be sufficient?

```suggestion
      if (maxMultiplier) {
```

Reply:
The above wouldn't support `maxMultiplier` being `0`, this may be more appropriate:

```suggestion
      if (maxMultiplier !== undefined) {
```

Reply:
Done.